### ISSUE TITLE
Line1 is incorrectly defined as link: A11y_mssql for VSCode_mssql Extension_Select top 1000 Results window_Usability

### ISSUE THREAD

**Issue by aniketr99** [microsoft/vscode-mssql#17877]
## GitHubTags:
#A11yTCS;#Win11;#DesktopApp;#BM_mssql for VSCode_Win32_Feb2024;#A11ySev4;#mssql for VSCode;#Win32;#A11yUsable;

## Environment Details:
Application Name: mssql for VSCode
Operating System: Win11[23H2]
OS Build: 26016.1012
Visual studio code: 1.86.2
Extension Version: v1.22.1

## Repro Steps:

1. Open Visual studio code
2. Tab till extension button and search for "mssql".
3. Tab till install and move to the SQL Server button.
4. Tab till Add Connection Button and insert the valid credentials.
5. Tab till the database and hit enter.
6. Tab till the System database and hit enter.
7. Tab till the Tables and hit enter.
8. Tab till any table under System tables and hit shift + F10
9. Tab till Select Top 1000 and hit enter
10. Tab till Execute Query and hit enter
11. Tab till message section and press enter.
12. Open Accessibility insights for windows and inspect line1 control.
13. Verify whether line1 control is incorrectly defined as link or not

## Actual:
Line1 is incorrectly defined as link.

## Expected:
Line1 should be defined as button.

## User Impact:
Screen reader user will not be able to get the shortcut key if screen reader does not announce it.

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

## Attachment:
![Screenshot (373)](https://github.com/microsoft/vscode-mssql/assets/100269580/1d2dacc7-391c-4f3b-8d0f-abc5b5012cc4)

**Comment by laurenastrid1** [comment]
Please make sure you use the latest version of VSCode mssql when testing this

**Comment by laurenastrid1** [comment]
This is not reproducible in the latest version of the extension: 

<img width="1911" height="1153" alt="Image" src="https://github.com/user-attachments/assets/fead6686-eded-4c0d-a0b9-7cacf31f8dfd" />

**Comment by laurenastrid1** [comment]
Closing because UI no longer exists.
